lib/geo/postgis.ex

```diff
@@ -28,7 +28,11 @@ if Code.ensure_loaded?(Ecto.Query) do
     end
 
     defmacro st_distance_sphere(geometryA, geometryB) do
-      quote do: fragment("ST_Distance_Sphere(?,?)", unquote(geometryA), unquote(geometryB))
+      quote do: fragment("ST_DistanceSphere(?,?)", unquote(geometryA), unquote(geometryB))
```

Comment:
I think we should leave the old one around for now and create a new macro. How long has `ST_DistanceSphere` been around? My only fear is someone using a version of postgis that doesn't have it.